cvrf2csaf/section_handlers/document_references.py

```diff
@@ -0,0 +1,25 @@
+
+from ..common.common import SectionHandler
+
+
+class DocumentReferences(SectionHandler):
+    def __init__(self):
+        super().__init__()
+
+    def _process_mandatory_elements(self, root_element):
+
+        references = []
+        if hasattr(root_element, 'Reference'):
+            for ref in root_element.Reference:
+                ref_csaf = dict()
+                ref_csaf['category'] = str(ref.attrib['Type'])
```

Comment:
If the `Type` is not defined this should result in `external` according to [CSAF-4.9.1-4](http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508893).

Reply:
Implemented.